tests/copp/test_copp.py

```diff
@@ -95,6 +101,51 @@ def test_no_policer(self, protocol, duthosts, rand_one_dut_hostname, ptfhost, co
                      copp_testbed,
                      dut_type)
 
+    @pytest.mark.parametrize("protocol", ["LACP",
+                                          "LLDP",
+                                          "UDLD",
+                                          "IP2ME"])
+    def test_counter(self, protocol, duthosts, rand_one_dut_hostname, ptfhost, copp_testbed, dut_type, counter_test):
+        duthost = duthosts[rand_one_dut_hostname]
+        trap_type = protocol.lower()
+
+        # wait until the trap counter is enabled
+        assert wait_until(10, 1, _check_trap_counter_enabled, duthost, trap_type), 'counter is not created for {}'.format(trap_type)
```

Comment:
The wait_until interface has been changed in PR #4516. This function call needs to be updated.

Reply:
Will do a rebase.